lca_website/ngo/views.py

```diff
@@ -32,4 +32,17 @@ def ngo_signup(request):
             template_data["errors"] = form.errors
             return HttpResponseRedirect("/ngo/signup")
     else:
-        return render(request, "ngo_signup.html")
+        return render(request, "ngo/signup.html")
+
+def index(request):
+    template_data['ngos'] = Ngo.objects.filter(approved=False)
+    return render(request, 'ngo/index.html')
+
+def approve(request, ngo_id):
```

Comment:
Approving should come under the admin app that has already been initialised and we also need a login interface to the admin side.